AggregateMetrics/AggregateMetrics/AzureWebApp/FlexiblePerformanceCounterGauge.cs

```diff
@@ -28,38 +44,43 @@ public string GetValue(string name, string json)
             return json.Substring(idx, endIdx - idx);
         }
 
+        /// <summary>
+        /// Main method to cache raw JSON and process it for raw output
+        /// </summary>
+        /// <param name="name"> Camel case name of sought counter</param>
+        /// <returns> Metric Telemetry object mt, with values for Name and Value </returns>
         public MetricTelemetry GetRawCounterValue(string name)
         {
             ObjectCache cache = MemoryCache.Default;
             CacheItemPolicy policy = new CacheItemPolicy();
             policy.AbsoluteExpiration = DateTimeOffset.Now.AddSeconds(5.0);
+            
+             /* http://remoteenvironmentvariables.azurewebsites.net/api/EnvironmentVariables/WEBSITE_COUNTERS_ASPNET/
+             http://remoteenvironmentvariables.azurewebsites.net/api/EnvironmentVariables/WEBSITE_COUNTERS_APP/
+             http://remoteenvironmentvariables.azurewebsites.net/api/EnvironmentVariables/WEBSITE_COUNTERS_CLR/
+             http://remoteenvironmentvariables.azurewebsites.net/api/EnvironmentVariables/WEBSITE_COUNTERS_ALL/ */
 
-            //http://remoteenvironmentvariables.azurewebsites.net/api/EnvironmentVariables/WEBSITE_COUNTERS_ASPNET/
-            //http://remoteenvironmentvariables.azurewebsites.net/api/EnvironmentVariables/WEBSITE_COUNTERS_APP/
-            //http://remoteenvironmentvariables.azurewebsites.net/api/EnvironmentVariables/WEBSITE_COUNTERS_CLR/
-            //http://remoteenvironmentvariables.azurewebsites.net/api/EnvironmentVariables/WEBSITE_COUNTERS_ALL/
-
-            if (cache["jsonContent"] == null)
+            if (cache[name] == null)
```

Comment:
@SergeyKanzhelev , is this the kind of separate class you were describing? It should be compatible with any counter that exists in a json. Right now the limitation is that the counter must come from the JSON returned from the HTTP Get from remoteenvironmentvariables